sdk/deviceregistry/arm-deviceregistry/review/arm-deviceregistry.api.md

```diff
@@ -292,205 +292,16 @@ export class DeviceRegistryManagementClient {
     readonly assetEndpointProfiles: AssetEndpointProfilesOperations;
     readonly assets: AssetsOperations;
     readonly billingContainers: BillingContainersOperations;
-    readonly discoveredAssetEndpointProfiles: DiscoveredAssetEndpointProfilesOperations;
```

Comment:
could you help check if this is _2 in code https://github.com/Azure/azure-sdk-for-js/blob/8dc1446b41c77fe43829d0095d9faee157dcceb6/sdk/deviceregistry/arm-deviceregistry/review/arm-deviceregistry.api.md?plain=1#L213?

Reply:
it's api-extractor's behaviour